lib/recipients/profile.js

```diff
@@ -34,7 +34,7 @@ module.exports = ({ schema, logger, task }) => {
     .then(applicant => {
       logger.debug(`applicant is ${applicant.firstName} ${applicant.lastName}`);
 
-      if (taskHelper.isAutoresolved(task) && action === 'update' && !initiatedByAsru) {
```

Comment:
The `action === 'update'` was removed because that's already filtered by `allowedActions` so was redundant.